subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1157,6 +1216,64 @@ static int oacp_read(struct bt_conn *conn,
 	return err;
 }
 
+static int oacp_write(struct bt_conn *conn, struct bt_otc_internal_instance_t *inst,
+		     const void *buf, uint32_t len, uint32_t offset,
```

Comment:
there's still some alignment issues. You should get [editorconfig](https://editorconfig.org/) or [clang-format](https://www.google.com/search?q=clang-format+plugins) plugins for your preferred editor for next time. Then settings are auto-applied per-repo.

Reply:
Somehow, clang-format is not working with my WSL and visual code.
I'll try to fix them next time.